Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcClient.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Blizzard.IO.Core.Rpc;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class NetqRabbitRpcClient : IRpcClient
+    {
+        private readonly INetqRabbitRpcConnection<Func<Type, object>> _netqRpcRabbitConnection;
+        private readonly ILogger _logger;
+
+        public NetqRabbitRpcClient(INetqRabbitRpcConnection<Func<Type, object>> netqRpcRabbitConnection, ILogger logger)
+        {
+            _logger = logger;
+            _netqRpcRabbitConnection = netqRpcRabbitConnection;
+        }
+
+        public TRespond Request<TRequest, TRespond>(TRequest request)
+        where TRequest : class
+        where TRespond : class
+        {
+            if (_netqRpcRabbitConnection.RpcMessageType == RpcMessageType.Abstract)
```

Comment:
I think that you switched by mistake the logics of `Abstract` and `Concrete`. This `if`'s logic should take of the `concrete` state not the `abstract`. Similarly, the `func<Type,object>` should be returned on the `abstract` state, not `concrete`

Reply:
I dont think you are correct (any way this code is about to change now with the change to the interface we did last night)